openreview/venue_request/process/revisionProcess.py

```diff
@@ -28,6 +28,11 @@ def process(client, note, invitation):
                 if matching_invitation:
                     matching_invitation.cdate = openreview.tools.datetime_millis(submission_deadline)
                     client.post_invitation(matching_invitation)
+                revision_invitation = openreview.tools.get_invitation(client, conference.get_invitation_id('Revision'))
+                if revision_invitation and conference.submission_stage.second_due_date:
```

Comment:
you are missing if now is between the first and second deadline.

The Revision invitation can be used later for authors to submit a new revision of the paper during the rebuttal or the camera ready version.

You can reproduce this calling the Revision stage when there is a Revision invitation happening that is not related to the second deadline.

For example here: https://github.com/openreview/openreview-py/blob/master/tests/test_venue_request.py#L1727

Reply:
Why do we need to compare with the first deadline? `now < second_deadline` should be sufficient, right?

Reply:
There's another issue here, if the second deadline is updated to sometime before now, it won't update the revision invitation.

Reply:
You are right, comparing the date with now is not correct. 

Another alternative is to update the Revision with the second_due_date only if there is no value for submission_revision_deadline. Because if there is a value, that means the PCs decided to start a revision stage and that should win the second_due_date.